sell-loot.lic

```diff
@@ -19,24 +19,45 @@ class SellLoot
 
     settings = get_settings
     town_data = get_data('town')
-    @actual_fucking_hometown = settings.hometown
+    @character_hometown = settings.hometown
     @hometown = town_data[settings.hometown]
     @bankbot_name = settings.bankbot_name
     @bankbot_deposit_threshold = settings.bankbot_deposit_threshold
     @bankbot_room_id = settings.bankbot_room_id
+    bankbot_enabled = settings.bankbot_enabled
+    @local_currency = town_data[@character_hometown]['currency']
 
     sell_gems("#{settings.gem_pouch_adjective} pouch") if settings.sell_loot_pouch
-
     check_spare_pouch(settings.spare_gem_pouch_container, settings.gem_pouch_adjective) if settings.spare_gem_pouch_container
 
     sell_bundle if settings.sell_loot_bundle
 
-    return if settings.sell_loot_skip_bank
-    exchange_coins
+    return if settings.sell_loot_skip_bank && !bankbot_enabled
+
+    @keep_coppers = parse(settings.sell_loot_money_on_hand.split(' '))
+    @keep_coppers = @bankbot_deposit_threshold if bankbot_enabled
+
+    if bankbot_enabled
+      if @bankbot_name.nil? || @bankbot_room_id.nil?
```

Comment:
Is this more appropriate to have in `validate.lic`?

Reply:
Possibly, there were checks to return if those settings were missing in the method give_money_to_bankbot.  I figured it would be more efficient to move them to the top so we don't go through any methods if they are missing.

Would you rather have me have exit the method w/o posting an error message?

Reply:
Or we can assume that if they have `bankbot_enabled`, they filled in both name and id?

Reply:
Yeah, I'm okay with that.